Resources/Locale/en-US/sticky/sticky-component.ftl

```diff
@@ -7,3 +7,8 @@ comp-sticky-success-unstick-bomb = You removed the bomb
 
 # General strings
 comp-sticky-unstick-verb-text = Unstick
+
+# Split stack fail message
+cannot-split-due-to-sticky = It is too sticky !
+cannot-merge-due-to-entity-stuck-on = It has things stuck to it !
+cannot-merge-due-to-entity-stuck = It is stuck !
```

Comment:
these are bad names for loc strings and should be more like the one above

Reply:
also, macoron's comment was not resolved.

Reply:
yeah I will think of a better name , and I didn't really understood what he meant by "in code any redundant whitespaces before punctuation marks"

Reply:
Macoron meant remove the space between the last word and the exclamation point for all the ones that have that.
"Something !"
to
"Something!"

Reply:
ah thank you for clarifying